pyproject.toml

```diff
@@ -28,6 +28,7 @@ pyyaml = "*"
 click = "*"
 python-json-logger = "^2.0.2"
 commonregex = "1.5.2"
```

Comment:
Can we drop the commonregex package or is it also still required?

Reply:
removed!